security-analysis/security-analysis-api/src/test/java/com/powsybl/security/detectors/LimitViolationDetectorForThreeWindingsTransformerTest.java

```diff
@@ -0,0 +1,281 @@
+/*
+ *  Copyright (c) 2017, RTE (http://www.rte-france.com)
+ *  This Source Code Form is subject to the terms of the Mozilla Public
+ *  License, v. 2.0. If a copy of the MPL was not distributed with this
+ *  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
```

Comment:
This header is not up to date:
```suggestion
/**
 * Copyright (c) 2023, RTE (http://www.rte-france.com)
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/.
 * SPDX-License-Identifier: MPL-2.0
 */
```